PhpcsChanged/CacheManager.php

```diff
@@ -163,6 +168,15 @@ public function addCacheEntry(CacheEntry $entry): void {
 		($this->debug)("Cache add: file '{$entry->path}', type '{$entry->type}', hash '{$entry->hash}', standard '{$entry->phpcsStandard}'");
 	}
 
+	public function getPhpcsStandardCacheKey( string $phpcsStandard, string $warningSeverity, string $errorSeverity ): string {
+		$warningSeverity = '' === $warningSeverity ? '5' : $warningSeverity;
```

Comment:
I think it might be worth creating a named variable (a class constant, maybe?) for `'5'` so it's clear that this is the phpcs default. That way it explains this block's behavior and also makes it easier if for some reason that number ever changes.

Reply:
Thank you so much for this! It's a great suggesting and it has reminded me why I love code reviews so much.